pkg/api/postage.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/ethersphere/bee/pkg/tracing"
```

Comment:
I commented on the other PR as well, you should be using gofmt. Please follow the coding style guide.